mscore/file.cpp

```diff
@@ -298,7 +299,8 @@ void MuseScore::openFiles(bool switchTab, bool singleFile)
              << tr("Bagpipe Music Writer Files (experimental)") + " (*.bww)"
              << tr("Guitar Pro Files") + " (*.gtp *.gp3 *.gp4 *.gp5 *.gpx *.gp)"
              << tr("Power Tab Editor Files (experimental)") + " (*.ptb)"
-             << tr("MuseScore Backup Files") + " (*.mscz, *.mscx,)";
+             << tr("MuseScore Backup Files") + " (*.mscz, *.mscx,)"
+             << tr("MuseScore Album Files") + " (*.msca *.album)";
```

Comment:
Are both <code>.msca</code> and <code>.album</code> identical? Or is one of them a compressed file?

Reply:
.msca is the new albums file. 
.album is the MuseScore2 album type.

They are same-ish, the new one has more information and is (in my opinion) more fitting. .album could be anything.